src/openloco/localisation/stringmgr.cpp

```diff
@@ -129,11 +129,13 @@ namespace openloco::stringmgr
         return (char*)regs.edi;
     }
 
-    static char* format_long_grouped(uint64_t value, char* buffer, uint8_t separator)
+    static char* format_int48_grouped(int64_t value, char* buffer, uint8_t separator)
```

Comment:
These change seem to have broken the finances window.
![screenshot_20180502_173201](https://user-images.githubusercontent.com/604665/39533193-cabdbe52-4e2e-11e8-8b33-71f5564a5e28.png)

Reply:
Do the original numbers match the original numbers? To me it's weird that it gets shifted by 31 bits now instead of 32.